### PR TITLE
Handling unable to capture game via window case

### PR DESCRIPTION
**Pull request by avoitenko-logitech**
### Description
Modified game capture module to handle the case when it is not possible to capture a window because it is not a game

### Motivation and Context
Added support of info type feature of text properties
Using both placeholder and compatibility info to notify user.
To refresh UI trick with custom flag was used.

### How Has This Been Tested?
Manually

### Types of changes
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: obs-studio-client/source/properties.hpp [hunk 1]
```diff
@@ -79,7 +79,14 @@ struct TextProperty : Property {
 		MULTILINE,
 	};
 
+	enum class InfoType {
```
Comment (summeroff) [inline obs-studio-client/source/properties.hpp]: it will be duplicate of InfoType in source/obs-property.hpp

File: source/obs-property.hpp [hunk 2]
```diff
@@ -127,7 +127,15 @@ struct TextProperty : Property {
 		Password,
 		MultiLine,
 	};
+
+	enum class InfoType : uint8_t {
```
Comment (summeroff) [inline source/obs-property.hpp:132]: As values will be mapped to ETextInfoType better to put values here too explicitly.

Comment (avoitenko-logitech) [reply source/obs-property.hpp:132]: Fixed

File: obs-studio-client/source/properties.hpp (hunk 1 above)
Comment (avoitenko-logitech) [reply obs-studio-client/source/properties.hpp]: I just followed existing approach here.
For the enum
```
	enum class Type {
		DEFAULT,
		PASSWORD,
		MULTILINE,
	};
```
there is a duplicate in `source/obs-property.hpp`
```
	enum class TextType : uint8_t {
		Default,
		Password,
		MultiLine,
	};
```

Comment (avoitenko-logitech) [reply obs-studio-client/source/properties.hpp]: Should I avoid a code duplication?

Comment (summeroff) [reply obs-studio-client/source/properties.hpp]: yes. if its possible better to not not have duplicated types.

Comment (avoitenko-logitech) [reply obs-studio-client/source/properties.hpp]: Fixed

**Review by summeroff — APPROVED** [review]

**Review by summeroff — APPROVED** [review]

**Pull request merged**